### ISSUE TITLE
Missing textures of tools and parts (I don't use optifine)

### ISSUE THREAD

**Issue by FearAK** [SlimeKnights/TinkersConstruct#4614]
### Minecraft Version

1.16.5

### Forge Version

36.2.2

### Mantle Version

1.6.123

### Tinkers' Construct Version

TConstruct-1.16.5-3.1.2.265

### Describe your issue

The tools and items are completely white no textures at all and I looked up possible issues and the only two i found were texfix and optifine which neither i have. I do however have Sodium Forge version, Idk if that's the problem or if there are some other mods i have that cause it?

PS. Yes I know my modlist is a clusterfk of mods..
![Screenshot_8](https://user-images.githubusercontent.com/46721025/131539067-d9f73b88-7312-43d4-8f11-7ba80fb57dcc.png)

1.16.5-StorageCabinet-1.0.jar
abnormals_core-1.16.5-3.2.0.jar
abnormals_delight-1.16.5-1.2.0.jar
abstoneedition-2.0.6-mc1.16.3-16.5.jar
additionalcolors-1.16.5-1.4.2.0-forge.jar
AdditionalEnchantedMiner-1.16.5-16.1.9.jar
additionallanterns-1.0.0-mc1.16.jar
advancedperipherals-1.16.5-0.7.1r.jar
AdvancementPlaques-1.16.5-1.4.0.jar
ae2ao-5.0.1-1.16.5.jar
ae2extras-1.3-1.16.5.jar
agricraft-3.0.5-hotfix.jar
AI-Improvements-1.16.2-0.3.0.jar
aiotbotania-1.16.5-1.8.1.jar
alexsmobs-1.12.0.jar
alltheores-1.3.6-1.16.5-36.1.0.jar
Ambience-1.16.5-1.0.0.jar
angelring-1.16.5-1.3.4.1.jar
Apotheosis-1.16.5-4.7.1.jar
appleskin-forge-mc1.16.x-2.1.0.jar
appliedenergistics2-8.4.0.jar
archers_paradox-1.16.5-1.3.1.jar
Architects-Palette-1.16.4-1.1.4.jar
architectury-1.22.32-forge.jar
ars_nouveau-1.16.5-1.22.3.jar
astralsorcery-1.16-1.16.5-1.13.12.jar
astrologicalsorcery-1.16.4-1.3.2.jar
AttributeFix-1.16.5-10.1.3.jar
AutoRegLib-1.6-49.jar
balancedenchanting-1.0.jar
BedBenefits-1.16.5-5.0.4.jar
BetterAdvancements-1.16.5-0.1.1.115.jar
BetterAnimationsCollection-v1.2b6-1.16.5.jar
BetterCaves-Forge-1.16.4-1.1.2.jar
BetterCrates-1.16.5-1.0.0.jar
BetterDungeons-1.16.4-1.2.jar
betterendforge-1.16.4-1.6.0.jar
BetterFurnacesReforged-1.16.5-1.2.jar
BetterHunger-1.16.5-6.0.2.jar
BetterMineshafts-Forge-1.16.4-2.0.4.jar
BetterStrongholds-1.16.4-1.2.1.jar
blockcarpentry-1.16-0.4.0.jar
BlockDiversity-1.16.5-1.3.2.jar
BloodMagic-1.16.4-3.1.1-16.jar
Bookshelf-Forge-1.16.5-10.2.27.jar
Botania-1.16.5-419.jar
BotaniaAdditionsMC1.16.4-1.0.4.jar
botanical-functioning-flora-1.0.0.jar
botanicalmachinery-1.16.4-0.4.1.jar
BotanyPots-1.16.5-7.1.23.jar
BotanyTrees-1.16.5-3.0.6.jar
BrandonsCore-1.16.5-3.0.8.232-universal.jar
brazier-3.0.9-forge.jar
buildersaddition-1.16.5-20210807a.jar
buildinggadgets-1.16.5-3.8.0.jar
bwncr-1.16.5-3.10.16.jar
cabletiers-1.16x-0.501.jar
caelus-forge-1.16.5-2.1.3.1.jar
cagedmobs-1.16.5-1.3.4.jar
camera-1.16.5-1.0.14.jar
cartographer-0.0.2-1.16.5-36.0.52.jar
castle_in_the_sky-1.16.5-0.2.6.jar
catalogue-1.3.0-1.16.5.jar
CBMultipart-1.16.5-3.0.2.119-universal.jar
cc-tweaked-1.16.5-1.98.2.jar
cfm-7.0.0pre22-1.16.3.jar
cgm-1.0.1-1.16.3.jar
champions-forge-1.16.5-2.0.1.7.jar
Chargers-1.16.5-3.1.0.5.jar
charm-forge-1.16.5-2.3.2.jar
ChickenChunks-1.16.5-2.8.0.86-universal.jar
chipped-1.1.2.jar
Chisel-MC1.16.5-2.0.1-alpha.4.jar
chiselsandbits-1.0.0-ALPHA-universal.jar
chocolate-1.3.0-1.16.4.jar
citadel-1.8.1-1.16.5.jar
CleanView-1.16.5-v2.jar
ClientTweaks_1.16.3-5.3.0.jar
cloth-config-4.11.26-forge.jar
Clumps-6.0.0.25.jar
CNB-1.16.3_5-1.2.11.jar
cobbler-1.6.1.jar
CodeChickenLib-1.16.5-4.0.3.432-universal.jar
cofh_core-1.16.5-1.3.1.jar
ColdEnchants2.0(Forge1.16.5)vrs1.0.6.jar
collective-1.16.5-2.58.jar
ColtCosmeticArmorMod-1.2.5.jar
colytra-forge-1.16.5-5.1.1.1.jar
comforts-forge-1.16.5-4.0.1.3.jar
compactmachines-4.0.0-beta.2.jar
CompactVoidMiners-B1.16.5-1.17.jar
concreteconversion-1.16.4-6.1.0.0-forge.jar
configurablebeacons-1.16.2-2.0.1.jar
configured-1.2.0-1.16.3.jar
connectedglass-1.1.0-mc1.16.jar
constructionwand-1.16.5-2.2.jar
Controlling-7.0.0.24.jar
cookiecore-1.16.2-3.0.2-9389769.jar
CookingForBlockheads_1.16.5-9.3.3.jar
CosmeticArmorReworked-1.16.5-v4.jar
CraftingTweaks_1.16.5-12.2.1.jar
create-addon-v5.jar
create-mc1.16.5_v0.3.2d.jar
create-stuff-additions1.16.5_v1.0.6.jar
createaddition-1.16.5-20210819a.jar
createdeco-1.0.2-1.16.5.jar
creativeapiary-1.16.5-1.7.jar
CreativeCore_v2.2.1_mc1.16.5.jar
creativecrafter-1.16x-0.152.jar
creativewirelesstransmitter-1.16x-1.11.jar
ctiers-1.16.5-1.35.jar
CTM-MC1.16.1-1.1.2.6.jar
Cucumber-1.16.4-4.1.10.jar
curios-forge-1.16.5-4.0.5.2.jar
cursedearth-3.2.jar
customizableelytra-1.16.4-1.5.5.jar
Cyclic-1.16.5-1.3.6.jar
CyclopsCore-1.16.5-1.11.9.jar
dankstorage-3.16.jar
DarkUtilities-1.16.5-8.0.11.jar
decorative_blocks-1.16.4-1.7.2.jar
DefaultOptions_1.16.5-12.2.1.jar
DeLogger-3.0.3-build.19+mc1.16.5.jar
demagnetize-forge-1.16.2-1.2.2.jar
deuf-1.16.4-1.1.jar
DiagonalFences-v1.1.1-1.16.5.jar
Dimensional-Paintings-1.16.5-1.1.1.jar
Ding-1.16.5-1.3.0.jar
DisenchantmentEditTable-1.16.5-1.0.3.jar
DoggyTalents-1.16.5-2.0.1.10.jar
DoubleSlabs-1.16-3.6.0.jar
Draconic-Evolution-1.16.5-3.0.11.396-universal.jar
drawerfps-2.1.jar
DungeonCrawl-1.16.5-2.3.2.jar
DustrialDecor-1.2.6.jar
DynamicSurroundings-1.16.4-4.0.4.2.jar
Easy+Shulker+Access-1.16.5-1.1.1.jar
EasyMagic-v1.0.2-1.16.5.jar
effortlessbuilding-1.16.3-2.23.jar
eidolon-0.2.7.jar
elevatorid-1.16.5-1.7.13.jar
embellishcraft-1.16.5-3.4.0.0.jar
emcbaubles-1.16.5-1.1.0.jar
enchantingcommands_1.16.5-1.7.jar
EnchantmentDescriptions-1.16.5-7.0.16.jar
endergetic-1.16.4-3.0.0.jar
EnderStorage-1.16.5-2.8.0.168-universal.jar
engineersdecor-1.16.4-1.1.14.jar
engineerstools-1.16.4-1.1.5.jar
EnigmaticGraves-1.6.1.jar
ensorcellation-1.16.5-1.3.1.jar
entangled-1.3.6-mc1.16.5.jar
EntityCulling-1.16.5-2.1.6.jar
entitydetectors-1.16.4-1.1.0.1.jar
envirocore-1.16.5-3.0.9.1.jar
enviroenergy-1.16.5-3.0.9.1.jar
enviromats-1.16.5-2.0.9.0.jar
envirotech-1.16.5-3.0.9.4.jar
Exchangers-1.16.5-3.0.2.jar
ExCompressum_1.16.5-4.0.4.jar
exlinefurniture-1.16.5.jar
ExNaturae-1.16.4-1.0.4.jar
exnihiloae-1.16-1.0.0.2.jar
exnihiloauto-1.16-1.2.0.1.jar
exnihilomekanism-1.16-1.0.0.1.jar
exnihilosequentia-1.16-20210826035238.jar
exnihilotinkers-1.16-1.0.0.0.jar
Exp Ore Mod - 1.16.5 - 1.3.jar
ExperienceBugFix-1.36.0.2.jar
ExplorersCompass-1.16.5-1.1.2-forge.jar
extendedbonemeal_1.16.5-1.4.jar
extradisks-1.16.4-1.5.1.jar
ExtraStorage-1.16.5-1.5.0.jar
ExtremeReactors2-1.16.5-2.0.27.jar
extremeSoundMuffler-3.13-1.16.5.jar
fakeblocks-1.16.5-1.0.2.jar
FarmersDelight-1.16.5-0.4.6.jar
farmersdelightintegrations-1.16.5-1.2.jar
FarmingForBlockheads_1.16.5-7.3.1.jar
FastFurnace-1.16.4-4.4.0.jar
FastLeafDecay-v25.jar
FastSuite-1.16.4-1.1.1.jar
FastWorkbench-1.16.4-4.5.1.jar
ferritecore-2.0.7-forge.jar
flickerfix-1.0.1.jar
FluxNetworks-1.16.5-6.1.7.12.jar
flywheel-1.16-0.2.3.jar
focusedbooks-0.2_hotfix.jar
forbidden_arcanus-16.2.0-beta-4.jar
ForgeEndertech-1.16.5-7.2.0.0-build.0177.jar
FpsReducer-forge-1.23-mc1.16.5.jar
FramedBlocks-2.9.3.jar
framedcompactdrawers-1.16-2.2.0.jar
frozenup-1.0.1.jar
ftb-backups-2.1.1.6.jar
ftb-essentials-1605.1.4-build.13.jar
ftb-library-forge-1605.3.2-build.64.jar
GameMenuModOption-1.16.4-1.8.jar
GameMenuRemoveGFARB-1.16.4-1.1.jar
gardentools-1.16.5-1.1.2.jar
geckolib-forge-1.16.5-3.0.45.jar
geode-1.1.2.jar
glassential-forge-1.16.5-1.1.7.jar
goblintraders-1.6.0-1.16.3.jar
goldenhopper-1.2.1-1.16.3.jar
growablecells-1.16.5-1.2.1.jar
GunpowderLib-1.16.5-1.2.2.jar
hats
Hats-1.16.5-10.3.1.jar
hole_filler_mod-1.2.4.2-mc_1.16.x-forge.jar
hourglass-1.16.5-1.1.1.2.jar
Iceberg-1.16.5-1.0.1.jar
iChunUtil-1.16.5-10.4.1.jar
illagers_plus-1.16.4v1.5.jar
immersivecooking-1.0.1.jar
ImmersiveEngineering-1.16.5-5.0.3-138.jar
immersiveposts-1.16.5-4.1.1.jar
industrial-foregoing-1.16.5-3.2.14.5-13.jar
infernal-expansion-2.2.0.jar
Infinitylib-2.0.6.jar
InventoryEssentials_1.16.5-2.3.2.jar
invtweaks-1.16.4-1.0.1.jar
ironchest-1.16.5-11.2.13.jar
IronJetpacks-1.16.5-4.2.1.jar
item-filters-forge-1605.2.5-build.9.jar
itemcollectors-1.1.2-mc1.16.5.jar
JAMD-1.2.1-build.13+mc1.16.5.jar
JAVD-2.6.1-build.48+mc1.16.5.jar
jei-1.16.5-7.7.1.121.jar
jei-professions-1.0.0-1.16.4.jar
jeiintegration_1.16.5-7.0.1.15.jar
JustEnoughCalculation-1.16.5-3.8.5.jar
JustEnoughResources-1.16.5-0.12.1.128.jar
Kiwi-1.16.5-3.5.2.jar
KleeSlabs_1.16.5-9.2.1.jar
Kobolds-1.4.5.jar
konkrete_1.1.6_MC_1.16.2-1.16.5.jar
Krate-1.16.5-0.1.4.jar
kubejs-forge-1605.3.15-build.100.jar
lanterncolors-1.16.3-2.1.1.jar
lazierae2-1.16.5-1.1.4.jar
lazydfu-0.1.3.jar
lemonlib-1.3.6.jar
LibX-1.16.3-1.0.76.jar
lightestlamp-4.0.7.jar
Lollipop-1.16.5-3.2.9.jar
lootbagmod-1.16.5-2.3.1.jar
LootingBowEnchantment-1.16.3-2.0.0.jar
malum-1.16.5-0.3.0.jar
mana-and-artifice-1.4.0.8.jar
mana-plus-1.2.0.0.jar
manaliquidizer-2.2.0.jar
mannequins-1.0.1-1.16.5-forge.jar
Mantle-1.16.5-1.6.123.jar
mapperbase-1.16.5-2.4.0.0.jar
matc-1.1.2.jar
materialis-1.16.5-2.1.1.jar
maxedvillagers-1.2.jar
mcjtylib-1.16-5.0.22.jar
mcw-bridges-1.0.6-mc1.16.5.jar
mcw-doors-1.0.3-mc1.16.5.jar
mcw-fences-1.0.0-mc1.16.5.jar
mcw-furniture-2.0.1-mc1.16.5.jar
mcw-lights-1.0.0-mc1.16.5.jar
mcw-paintings-1.0.2-mc1.16.5.jar
mcw-roofs-2.0.1-mc1.16.5-4.jar
mcw-trapdoors-1.0.2-mc1.16.5.jar
mcw-windows-2.0.0-mc1.16.5.jar
mdecore-1.16.4-16.1.0.jar
Measurements-1.16.5-1.1.jar
Mekanism-1.16.5-10.0.21.448.jar
MekanismAdditions-1.16.5-10.0.21.448.jar
MekanismGenerators-1.16.5-10.0.21.448.jar
MekanismTools-1.16.5-10.0.21.448.jar
mgui-1.16.5-3.2.0.jar
mightyarchitect-mc1.16.3_v0.5.jar
minecolonies-0.14.353-BETA-universal.jar
mininggadgets-1.7.5.jar
miniutilities-1.2.1.jar
MobTracker(Forge1.16.4)vrs1.0.2.jar
mob_grinding_utils-1.16.5-0.4.28.jar
modnametooltip_1.16.2-1.15.0.jar
mods.txt
modular-routers-1.16.5-7.5.1-69.jar
ModularPowersuits-1.16.4-2.0.4.jar
MoreEnchantments_MC_1.16.4_Vers.1.3.1.jar
moreoverlays-1.18.15-mc1.16.5.jar
morered-1.16.5-2.1.1.0.jar
morevanillalib-1.16.4-1.4.1.jar
morevillagers-FORGE-1.16.5-1.5.4.jar
Morph-o-Tool-1.4-27.jar
mostructures-forge-1.2.0-1.16.5.jar
MouseTweaks-2.14-mc1.16.2.jar
movingelevators-1.2.34-mc1.16.5.jar
mowziesmobs-1.5.19.jar
mrplibrary-1.16.5-2.0.1.0-forge.jar
multibeds-1.16-1.7.9.jar
MysticalAdaptations-1.16.5-1.2.1.jar
MysticalAgradditions-1.16.5-4.2.0.jar
MysticalAgriculture-1.16.5-4.2.1.jar
mysticalworld-1.16.5-0.3.2.18.jar
MythicBotany-1.16.5-1.4.1.jar
NaturesCompass-1.16.5-1.9.1-forge.jar
NetherPortalFix_1.16.3-7.2.1.jar
nethers_delight-2.1.jar
nomowanderer_MC1.16.5_1.2.jar
notenoughcrashes-3.2.0-forge.jar
Notes-1.16.5-1.2.3.jar
novillagerdm-1.0.1.jar
nuclear_craft-1.16.5-0.9.3.jar
Numina-1.16.4-2.0.10.jar
obfuscate-0.6.2-1.16.3.jar
observerlib-1.16.5-1.5.3.jar
occultism-1.16.5-1.10.0.jar
OreExcavation-1.8.157.jar
OuterEnd-0.2.14.jar
overloadedarmorbar-5.1.0.jar
paintings-1.16.4-7.0.0.1.jar
passiveendermen_1.16.5-1.5.jar
Patchouli-1.16.4-53.1.jar
pipez-1.16.5-1.2.8.jar
pitg-1.16.5-2.0.1.jar
Placebo-1.16.5-4.6.0.jar
plonk-1.16.5-9.0.8.jar
pneumaticcraft-repressurized-1.16.5-2.14.1-245.jar
polymorph-forge-1.16.5-0.25.jar
portable_stonecutter-1.16.4-1.3.4.jar
potiondescriptions-1.16.5-1.5.3.jar
Powah-1.16.5-2.3.16.jar
prefab-1.6.5.10.jar
PrettyPipes-1.9.4.jar
ProjectE-1.16.5-PE1.0.1B.jar
ProjectExtended-1.16.5-1.2.0.jar
ProjectRed-1.16.5-4.10.0-core.jar
ProjectRed-1.16.5-4.10.0-exploration.jar
ProjectRed-1.16.5-4.10.0-illumination.jar
ProjectRed-1.16.5-4.10.0-integration.jar
ProjectRed-1.16.5-4.10.0-transmission.jar
Psi 1.16-95.jar
psipherals-1.4.1.jar
PuzzlesLib-v1.0.10-1.16.5.jar
QuantumStorage-1.16.4-5.3.0.jar
Quark-r2.4-316.jar
quartz-chests-1603.1.0.8.jar
randompatches-2.4.4-forge.jar
rangedpumps-0.8.2.jar
refinedpipes-0.5.jar
refinedstorage-1.9.15.jar
refinedstorageaddons-0.7.3.jar
relics-0.2.6.jar
repurposed_structures_forge-3.2.2+1.16.5.jar
ResourcefulBees-1.16.5-0.9.9.3.jar
rftoolsbase-1.16-2.0.11.jar
rftoolsbuilder-1.16-3.1.2.jar
rftoolscontrol-1.16-4.0.11.jar
rftoolsdim-1.16-7.0.12-beta.jar
rftoolspower-1.16-3.0.9.jar
rftoolsstorage-1.16-2.0.13.jar
rftoolsutility-1.16-3.1.2.jar
rhino-1605.1.2-build.48.jar
rsgauges-1.16.4-1.2.11.jar
rsrequestify-1.16.5-2.1.3.jar
Runelic-1.16.5-7.0.2.jar
savageandravage-1.16.5-3.1.0.jar
SaveMyStronghold-1.16.4-1.0.jar
ScalableCatsForce-2.13.5-build-2-with-library.jar
Scannable-MC1.16.5-Forge-1.7.4+21.jar
Scorge-3.1.3.jar
scythd-1.16.5-0.1.9.jar
selene-1.16.5-1.6.1.jar
shetiphiancore-1.16-3.8.5.jar
ShoulderSurfing-1.16.5-2.2.1.jar
simplemagnets-1.1.2-mc1.16.5.jar
simplyutilities-1.16.5-0.5.1-build.13.jar
skinned_lanterns-1.16.5-1.3.1.jar
smoothboot-forge-1.16.4-1.2.2.jar
snad-1.0.9-forge.jar
sodium-forge-2.9.10+rev.ea6353c.jar
SolarFluxReborn-1.16.3-16.2.5.jar
sophisticatedbackpacks-1.16.5-3.2.7.326.jar
SoundFilters-0.14_for_1.16.2.jar
SpawnerFix-1.16.2-1.0.0.2.jar
stalwart-dungeons-1.16.5-1.1.2.jar
Statues-1.16.5-0.1.7.3.jar
StorageDrawers-1.16.3-8.3.0.jar
storageracks-1.6.jar
structurescompass-1.16.5-1.2.0.jar
StructuresPlusEnd.jar
structure_gel-1.16.5-1.7.7.jar
structurize-0.13.210-ALPHA-universal.jar
supermartijn642configlib-1.0.9-mc1.16.jar
supermartijn642corelib-1.0.12-mc1.16.5.jar
supplementaries-1.16.5-0.15.2b.jar
swingthroughgrass-1.16.4-1.5.3.jar
tanknull-2.3-1.16.4.jar
tconplanner-1.16.5-1.0.0.jar
TConstruct-1.16.5-3.1.2.265.jar
TerraForged-1.16.5-0.2.14.jar
theoneprobe-1.16-3.1.4.jar
thermal_cultivation-1.16.5-1.3.0.jar
thermal_expansion-1.16.5-1.3.0.jar
thermal_foundation-1.16.5-1.3.2.jar
thermal_innovation-1.16.5-1.3.0.jar
thermal_locomotion-1.16.5-1.3.0.jar
time-in-a-bottle-1.1.0.jar
timeless_and_classic_4.0.0_RELEASE.jar
tinyredstone-1.16.5-1.9.1.jar
titanium-1.16.5-3.2.8.5-19.jar
TMechworks-1.16.3+-2.2.5.jar
TNT_Ultra_V2.5.jar
Toast-Control-1.16.4-4.3.1.jar
tomeofblood-1.16.5-1.1.jar
ToolBelt-1.16.5-1.16.0.jar
topaddons-1.16.5-2.1.5-beta.jar
torcherino-14.0.0+1.16.4.jar
torchmaster-2.3.8.jar
torohealth-1.16.4-forge-4.jar
toughnessbar-6.1.jar
TrampleStopper-2.6.1-build.22+mc1.16.5.jar
Translocators-1.16.5-2.6.0.82-universal.jar
trashcans-1.0.10-mc1.16.5.jar
TrashSlot_1.16.3-12.2.1.jar
TravelAnchors-2.4.jar
TravelersTitles-1.16.4-1.5.jar
twilightforest-1.16.5-4.0.546-universal.jar
UndeadExpansion-1.16.5-2.0.0.jar
UNDEADv.1.2a.release.jar
upgrade_aquatic-1.16.5-3.1.0.jar
upsizer-mc1.16.4-6.1.1.jar
u_team_core-1.16.5-3.2.1.196.jar
valhelsia_core-16.0.10.jar
valkyrielib-1.16.5-3.0.9.2.jar
villagertools-1.16.5-1.0.1.jar
Waystones_1.16.5-7.6.3.jar
windowlogging-mc1.16.3_v0.0.4.jar
wirelesschargers-1.0.0a-mc1.16.jar
woot-1.16.5-1.0.8.0.jar
wormhole-1.1.8-mc1.16.5.jar
XaerosWorldMap_1.17.0.1_Forge_1.16.5.jar
Xaeros_Minimap_21.16.0_Forge_1.16.5.jar
xnet-1.16-3.0.13.jar
XNetGases-1.16.5-2.3.7.jar
xptome-1.16.5-v2.1.2.jar
YungsApi-1.16.4-Forge-12.jar
YungsExtras-Forge-1.16.4-1.0.jar
ZeroCore2-1.16.5-2.1.3.jar
[1.16.4]+MassInscriber-0.1.jar

### Reproduced with only Tinkers' Construct?

No

### 

- [X] Confirm you have checked the roadmap, what's new, and the FAQ for your issue
- [X] Confirm you are not using performance enhancers, such as optifine

**Comment by Insane96** [comment]
> Confirm you are not using performance enhancers, such as optifine
 
 You lied, you even stated you're using Sodium.

**Comment by KnightMiner** [comment]
The issue template is not that hard to follow. It tells you to take the minimal steps to debug this issue, including telling you to ideally narrow down the mod list and get a list of fewer than 10 mods. I can do nothing with 100+ mods, and telling me you know its a problem does not mean I can suddenly do more. At minimum I expect you to test without performance enhancing mods, but you choose to instead lie and say you are using none despite mentioning Sodium above..

In this case I can save you time by saying your performance enhancing mods are to blame, this is a bug with Sodium, see spoorn/sodium-forge#163. Closing as I cannot do anything about performance enhancers that disable the forge lighting pipeline.
